### ISSUE TITLE
What ip or users are blocked ?

### ISSUE THREAD

**Issue by mohammadrabetian** [jsocol/django-ratelimit#210]
I want to check what ip or users (any given key) has exceeded the ratelimit on my apis. I would have liked if django-ratelimit had that option -> for example a function which returns blocked ips, remaining block time as a dictionory. 

I would like to work on it and send a PR. Is that ok ?

**Comment by jsocol** [comment]
Paraphrasing just so I’m sure I got you: you want some way to list the users who are currently over their rate limit, right?

That’s definitely an interesting but of functionality. How are you thinking of implementing it? Because of how ratelimit works internally, there’s no list you can easily query (it relies on one-way hash functions with periodically changing inputs, and this is important for both speed and security).

I can definitely also see wanting to know which users had historically exceeded the limits, not just current.

**Comment by mohammadrabetian** [comment]
Exactly! I wanted to make sure you have a positive opinion about this. Now I'm gonna think about it and come up with a solution in a day or two. 
I figure it's easier to just show you in code and have a discussion on it rather than to describe how I'm gonna implement it.
